### PR TITLE
Port extract values and extract values as dict ROS2

### PR DESCRIPTION
**Pull request by elvioaruta**
- Ported both functions
- attr_getter is python3 library and the code doesn't need to be changed
- Only missing to find a replacement for genpy

### CODE REVIEW

File: inorbit_republisher/inorbit_republisher/republisher.py [hunk 1]
```diff
@@ -173,38 +173,44 @@ def callback(msg, repub=repub):
...
+    return val
+
+
+"""
+Extracts several values from a given nested msg field and returns a dictionary of
+<field, value> elements
+"""
+
+
+def extract_values_as_dict(msg, mapping, node):
```
Comment (elvioaruta) [inline inorbit_republisher/inorbit_republisher/republisher.py:197]: I don't like passing the node to this function just to have the ability to log inside, fyi I'm aware of this ugly code

Comment (adamantivm) [reply inorbit_republisher/inorbit_republisher/republisher.py:197]: I agree. Can you add a quick TODO? This will be solved when we use the class format

**Comment by adamantivm** [comment]
LGTM

File: inorbit_republisher/inorbit_republisher/republisher.py (hunk 1 above)
Comment (elvioaruta) [reply inorbit_republisher/inorbit_republisher/republisher.py:197]: Done

**Pull request merged**